### PR TITLE
CT-1402 fix2 token management endpoint

### PR DESCRIPTION
**Pull request by vojtabiberle**
Jira: [CT-1402](https://keboola.atlassian.net/browse/CT-1402)
Connection PR: https://github.com/keboola/connection/pull/5052

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

## Release

- [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

---

[CT-1402]: https://keboola.atlassian.net/browse/CT-1402?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/ProjectsTest.php [hunk 1]
```diff
@@ -1176,8 +1177,29 @@ public function testCreateProjectStorageTokenWithMangeTokensPermission()
         $verified = $client->verifyToken();
         $this->assertEquals($project['id'], $verified['owner']['id']);
         $this->assertTrue($verified['canManageBuckets']);
-        $this->assertTrue($verified['canManageTokens']);
+        $this->assertFalse($verified['canManageTokens']); // we do not support canManageTokens in storage tokens
```
Comment (tomasfejfar) [inline tests/ProjectsTest.php:1180]: True musí dál fungovat :(

Comment (tomasfejfar) [reply tests/ProjectsTest.php:1180]: (dokud nevyčistíme deprecations)

**Review by tomasfejfar — CHANGES_REQUESTED** [review]

File: tests/ProjectsTest.php (hunk 1 above)
Comment (vojtabiberle) [reply tests/ProjectsTest.php]: jj, fixnuto

File: openapi.yml [hunk 2]
```diff
@@ -854,7 +854,11 @@ paths:
                     created: '2014-11-11T08:40:51.620Z'
       summary: Retrieve an organization
       operationId: Retrieve an organization
-      description: ''
+      description: >-
```
[the remark this answers is not in the document]
Comment (vojtabiberle) [reply openapi.yml:857]: to udělal generátor :-)

File: tests/ProjectDeleteTest.php [hunk 3]
```diff
@@ -149,7 +149,6 @@ public function testPurgeExpiredProjectRemoveMetadata()
         $params = [
             'canManageBuckets' => true,
             'canReadAllFileUploads' => true,
-            'canManageTokens' => true,
```
Comment (tomasfejfar) [inline tests/ProjectDeleteTest.php:152]: Zkusils to pustit? Možná to neprojde :D

Comment (vojtabiberle) [reply tests/ProjectDeleteTest.php:152]: Zkoušel a na lokále mi to prochází.

**Review by tomasfejfar — APPROVED** [review]

**Pull request merged**